src/gs-category-page.c

```diff
@@ -280,6 +280,16 @@ gs_category_page_create_filter_list (GsCategoryPage *self,
 			g_debug ("not showing %s/%s as no apps",
 				 gs_category_get_id (category),
 				 gs_category_get_id (s));
+
+			// re-filter USB category with no apps so the
+			// placeholder app tiles get cleared out, then set
+			// "empty state" message for an empty USB disk
+			if (g_strcmp0 (gs_category_get_id (category), "usb") == 0) {
+				gs_category_page_populate_filtered (self, s);
+
+				// TODO: show a "No apps available on USB" message
```

Comment:
Use a C-style comment, and add the comment above the function call.

Reply:
Fixed locally (though this will disappear when implemented)